api/TaskAttributionTiming.json

```diff
@@ -14,10 +15,13 @@
             "version_added": null
           },
           "firefox": {
-            "version_added": null
+            "version_added": false,
+            "notes": "See <a href='https://bugzil.la/1348405'>bug 1348405</a>."
+
```

Comment:
The blank line here is failing some of the automated checks. I expect that removing this line will fix it